piptools/scripts/options.py

```diff
@@ -364,3 +370,28 @@ def _get_default_option(option_name: str) -> Any:
     is_flag=True,
     help="Restrict attention to user directory",
 )
+
+build_deps_for = click.option(
+    "--build-deps-for",
+    "build_deps_for_distributions",
+    multiple=True,
+    type=click.Choice(ALL_BUILD_DISTRIBUTIONS),
+    help="Name of a distribution to install build dependencies for. "
+    "Static dependencies declared in 'pyproject.toml::build-system.requires' will be included as "
+    "well; may be used more than once.",
+)
+
+all_build_deps = click.option(
+    "--all-build-deps",
+    is_flag=True,
+    default=False,
+    help="Install build dependencies for all distributions. "
```

Comment:
```suggestion
    help="Install each build environment's dependencies for all distributions. "
```

Reply:
Waiting for https://github.com/jazzband/pip-tools/pull/1681/files#r1299461192 to reach a conclusion

Reply:
> Waiting for https://github.com/jazzband/pip-tools/pull/1681/files#r1299461192 to reach a conclusion

Suggested using "build target" in https://github.com/jazzband/pip-tools/pull/1681/files#r1328025077. WDYT?